### ISSUE TITLE
Icon request: GNU

### ISSUE THREAD

**Issue by shield-9** [FortAwesome/Font-Awesome#2665]
I would like to request the GNU Logo.
http://www.gnu.org/graphics/agnuhead.en.html

**Comment by christian-momon** [comment]
Expected response :disappointed:

**Comment by christian-momon** [comment]
To improve submission process, I create a separated issue for my suggested image (#6040).

**Comment by christian-momon** [comment]
Seven month later: no anwser, nothing new, void. So, what is the problem?

**Comment by shield-9** [comment]
@tagliala Any updates on this request?

**Comment by tagliala** [comment]
@shield-9 sorry, no updates here

@christian-momon you could consider a custom build https://github.com/FortAwesome/Font-Awesome/wiki/Customize-Font-Awesome and share your repo here. Unfortunately there is nothing you can do to speed up the process

**Comment by 2mac** [comment]
[This logo](http://www.gnu.org/graphics/gnu-head-30-years-anniversary.svg) would better fit the flat style, I think.

**Comment by fauno** [comment]
:+1:

**Comment by eduoda** [comment]
+1

**Comment by valioDOTch** [comment]
+1

**Comment by miguelmenendez** [comment]
+1

**Comment by mizerydearia** [comment]
+1

**Comment by Petrouchka** [comment]
+1

**Comment by Angristan** [comment]
+1

**Comment by cuerdo** [comment]
+1

**Comment by starfys** [comment]
+1 This would be useful for webpages and projects promoting free software

**Comment by fauno** [comment]
would you accept PRs on this issue? maybe i can get someone to do it :)

**Comment by fauno** [comment]
well, apparently not

http://fontawesome.io/community/#submitting-pull-requests

**Comment by Berkmann18** [comment]
+1

**Comment by Petrouchka** [comment]
No more need to vote, see https://forkawesome.github.io/Fork-Awesome/

**Comment by Berkmann18** [comment]
@Petrouchka There's no GNU icon on that set.

**Comment by Petrouchka** [comment]
But you can ask one. On FontAwesome, you can wait for 100 years.
"Since Oct 22, 2016, Font Awesome has frozen all pull requests proposing new icons. With Fork Awesome, we are welcoming all your pull requests."

**Comment by Berkmann18** [comment]
@Petrouchka Fair enough, your message comes across as "Ay here's this site which has what you need" and sadly don't.

**Comment by Berkmann18** [comment]
@Petrouchka No, but I'm doing it now with the icons that aren't in FA.
And by the way, it seems that some icons were implemented here but not on Fork Awesome (e.g: Vue).

**Comment by sudoman** [comment]
+1

**Comment by SidharthArya** [comment]
+1
I don't get this. The very community which is responsible for open source isn't in the icon set. Sure we can make one icon on our own but it seems like the font awesome community isn't interested in backing up the free software foundation in the slightest way they can. 
Given the contribution of the free software foundation, they deserve it.

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
